src/index.tsx

```diff
@@ -46,6 +47,33 @@ type EventLog = {
   timestamp: Date;
 };
 
+type verifyServExtProps = {
```

Comment:
Can you please spell out `Server` and `Extension`? Space isn't super tight here, and this helps with accessibility, in case someone is using a screen reader or translation software to read out code.

Reply:
That's an interesting point about accessibility, have not considered it before. Will implement the suggestion, will look more into negative impacts of using abbreviations in code as I quite like using them.